packages/next-ui/Navigation/components/NavigationFab.tsx

```diff
@@ -54,19 +59,20 @@ export function NavigationFab(props: NavigationFabProps) {
     >
       <MotionDiv
         className={classes.wrapper}
-        sx={(theme) => ({
+        sx={{
           [theme.breakpoints.down('md')]: {
             opacity: '1 !important',
             transform: 'none !important',
           },
-        })}
+        }}
         style={{ opacity }}
       >
         <Fab
+          tabIndex={!isMobile && !fabIsVisible ? -1 : 0}
```

Comment:
Why does this behaviour need to be different on mobile & desktop?

Reply:
Because the menu fab is initially hidden until the user scrolls a bit. So it would be better if it was only focusable when it's actually visible according to general WCAG. However in this case i think it would be better to just leave it as-is. As fiddling about with this might create performance issues And the button itself remains useable even when not yet scrolled.

Most of the time the user will be interacting with the page though, so I've removed this check and just left the menu button in the normal tab flow.